### ISSUE TITLE
How to save Fine-Tuned Model

### ISSUE THREAD

**Issue by ed-fish** [hushell/pmf_cvpr22#12]
How did you save and export a fine-tuned model on specific meta-tasks?

It appears that during the fine-tuning stage the weights of the model are reset to the meta-training weights for each episode. The average accuracy is then reported by taking the average of the acc from all the training episode. 

If I would like to save the model after the fine-tuning stage - should I average the weights from all the fine-tuning episodes? 

Thanks for your help.

**Comment by hushell** [comment]
I think there is a bit misunderstanding on fine-tuning here. The goal of few-shot learning is to learn a model that can be quickly deployed on a new task/episode (with fine-tuning on the support set of the task). So fine-tuning is not about learning a globally shared model among many tasks anymore, it's about quickly adapting a pre-trained and/or meta-trained model (which is class-agnostic) to be a task-specific model. Note that fine-tuning on a particular task makes use of the support set of that task only. You can save models for different episodes but it probably doesn't make sense to do so.

**Comment by ArlixLin** [comment]
Hi! I have the same question: why can't I find the fine-tuning code?
According to your answer, did you mean that: "fine-tuning = simply changing the support image to the object scene"?
Looking forward to your answer! Thanks a lot.

**Comment by ArlixLin** [comment]
@hushell adding an at
